### DISCUSSION TITLE
Questions?

### DISCUSSION THREAD

**Discussion by Leo3213** [Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3#5] [General]
This is where people have questions about what works and doesnt

**Comment by robi62** [comment]
Hi I by luck found your github wonderful so I can boot to windows to
My hp is a 430 g3 which is almost identical The only problem i have is that I can only boot in verbose If i do a normal boot does not do anything Any ideas???
Regards
Roberto

**Reply by Leo3213** [reply]
I wouldn't really know but if you find out let me know 😇
